packages/features/bookings/Booker/components/LargeCalendar.tsx

```diff
@@ -37,10 +37,14 @@ export const LargeCalendar = ({
     if (!schedule.slots) return availableTimeslots;
 
     for (const day in schedule.slots) {
-      availableTimeslots[day] = schedule.slots[day].map((slot) => ({
-        start: dayjs(slot.time).toDate(),
-        end: dayjs(slot.time).add(eventDuration, "minutes").toDate(),
-      }));
+      availableTimeslots[day] = schedule.slots[day].map((slot) => {
+        const { time, ...rest } = slot;
```

Comment:
Here I'm allowing more props to be passed in slot. This should work for a long run since we now have defined our UI implementations. Later down we can explore app dir approach for this component and be served with what the UI need directly from the server.